spec/language/Ch02_Core_Concepts.md

```
@@ -210,7 +210,7 @@ The `alias` construct may expose internal nets as top-level ports to improve des
 
 ## 2.8 Structural Composition Primitives
 
-#### Schematic-like sugar (all expand to primitives in CasIR)
+#### Schematic-like sugar (all expand to primitives in ACIR)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)